activesupport/lib/active_support/cache/redis_cache_store.rb

```diff
@@ -5,16 +5,10 @@
   require "redis"
   require "redis/distributed"
 rescue LoadError
-  warn "The Redis cache store requires the redis gem, version 4.0.1 or later. Please add it to your Gemfile: `gem \"redis\", \"~> 4.0\"`"
+  warn "The Redis cache store requires the redis gem, version 4.0.1 or later. Please add it to your Gemfile: `gem \"redis\", \">= 4.0.1\"`"
   raise
 end
 
-# Prefer the hiredis driver but don't require it.
-begin
-  require "redis/connection/hiredis"
```

Comment:
This require is a bit pointless because it's automatic if they have `gem "hiredis"` in their Gemfile.